frontend/src/component/admin/auth/AuthSettings.tsx

```diff
@@ -51,44 +53,16 @@ export const AuthSettings = () => {
     return (
         <div>
             <PermissionGuard permissions={[ADMIN, UPDATE_AUTH_CONFIGURATION]}>
-                <PageContent
-                    withTabs
-                    header={
-                        <Tabs
-                            value={activeTab}
-                            onChange={(_, tabId) => {
-                                setActiveTab(tabId);
-                            }}
-                            indicatorColor='primary'
-                            textColor='primary'
-                        >
-                            {tabs.map((tab, index) => (
-                                <Tab
-                                    key={`${tab.label}_${index}`}
-                                    label={tab.label}
-                                    id={`tab-${index}`}
-                                    aria-controls={`tabpanel-${index}`}
-                                    sx={{
-                                        minWidth: {
-                                            lg: 160,
-                                        },
-                                    }}
-                                />
-                            ))}
-                        </Tabs>
-                    }
-                >
-                    <div>
-                        {tabs.map((tab, index) => (
-                            <TabPanel
-                                key={index}
-                                value={activeTab}
-                                index={index}
-                            >
-                                {tab.component}
-                            </TabPanel>
-                        ))}
-                    </div>
+                <PageContent header={activeTab}>
+                    <Routes>
+                        <Route path='/' element={<OidcAuth />} />
+                        <Route path='/saml' element={<SamlAuth />} />
+                        <Route path='/password' element={<PasswordAuth />} />
+                        {googleAuthEnabled && (
+                            <Route path='/google' element={<GoogleAuth />} />
+                        )}
+                        <Route path='/scim' element={<ScimSettings />} />
+                    </Routes>
```

Comment:
We moved the old way of doing this to OldAuthSettings.tsx. This allows us to link directly to a SSO type from the menu. The old interface will be removed when we remove the flag